### ISSUE TITLE
[v1] Editor flickers when changing images

### ISSUE THREAD

**Issue by bedeoverend** [SimplaElements/simpla-img#45]
The editor flickers when changing between items of different UID. This is most likely due to the complex observers being using the `image-handler` behavior. Instead of these observers, we should manually fire / listen to events so that the editor changes it's src at the correct time, and ensures it's hidden until that point.

**Comment by seaneking** [comment]
FWIW I've never been able to repro this - is it still an issue with 1.0?

**Comment by bedeoverend** [comment]
@seaneking yeah I'm still getting this occasionally. Try it with the demo page - upload an image into one, then go to another image to edit / go back and forth between two images. That's a pretty consistent repro for me. It'll also sometimes carry over the cached image I believe.
